examples/mnist/.tensorleap/dataset.py

```diff
@@ -27,25 +27,24 @@
 
 
 def preprocess(df: pd.DataFrame) -> Tuple[np.ndarray, np.ndarray]:
+    print('preprocessing the data')
     data_X = df.drop('label', axis=1).to_numpy()
-    data_X = np.reshape(data_X, (len(data_X), 28, 28, 1)) / 255.  # normalize to range 0-1
+    data_X = np.reshape(data_X, (len(data_X), IMAGE_DIM, IMAGE_DIM, 1)) / 255.  # normalize to range 0-1
```

Comment:
Consider using:
```
    data_X = np.expand_dims(data_X, axis=-1)  # Reshape :,28,28 -> :,28,28,1
    data_X = data_X / 255                                     # Normalize to [0,1]
```
This way you could drop the `IMAGE_DIM` variable as well.

Reply:
It might be more convenient to show the image dim by keeping the variable. What do you think?